Makefile

```diff
@@ -398,10 +403,10 @@ geoportal/c2cgeoportal_geoportal/locale/c2cgeoportal_geoportal.pot: \
 	mkdir --parent $(dir $@)
 	pot-create --keyword _ --config $< --output $@ $(SRC_FILES)
 
-admin/c2cgeoportal_admin/locale/c2cgeoportal_admin.pot: lingua.cfg
+admin/c2cgeoportal_admin/locale/c2cgeoportal_admin.pot: admin/lingua.cfg $(ADMIN_SRC_FILES)
 	$(PRERULE_CMD)
 	mkdir --parent $(dir $@)
-	pot-create --keyword _ --config $< --output $@ $(SRC_FILES)
+	pot-create --config $< --output $@ $(ADMIN_SRC_FILES)
```

Comment:
Can't we share the same config file?

Reply:
We could, but we may have different options, geoportal do not use jinja2 for example.

Reply:
Yes but we can add it for booth?
In general I prefer to have on file it it stay simple :-)